src/passes/SignExtLowering.cpp

```diff
@@ -68,6 +68,14 @@ struct SignExtLowering : public WalkerPass<PostWalker<SignExtLowering>> {
       }
     }
   }
+
+  void run(Module* module) override {
+    if (!module->features.has(FeatureSet::SignExt)) {
```

Comment:
Is it possible the module could contain sign ext instructions but this feature not be enabled?  (I guess in that case it would not validate?)

Reply:
I also wonder why the test we have in emscripten for this was passing.. investigating.

Reply:
The emscripten test merely checks that `--signext-lowering` is part of the wasm-opt command line .. I didn't both to actually decompile the module to verify binaryen was actually doing the right thing.